### ISSUE TITLE
Create functions that pass a throwable to the underlying logging system

### ISSUE THREAD

**Issue by jadlr** [zio/zio-logging#39]
I'd like to be able to pass a `Throwable` to the logging system. 

Something like this would be possible:
```scala
  someAction.foldM(
    { error =>
      log.error(s"Failed to do the action", error) *> InternalServerError()
    }, { response =>
      Ok(response)
    })
```

Thanks!

**Comment by jdegoes** [comment]
We should add the more fundamental one in ZIO, which is logging a `Cause`, and then a more special case one can be implemented in terms of that one, e.g.:

```scala
def error(cause: Cause[Any]): UIO[Unit]

def throwable(t: Throwable): UIO[Unit] = ...
```

**Comment by pshemass** [comment]
@jdegoes I would like to add methods in package objects for `info` and `debug` as shortcuts to `log(LogLevel.Info)` and `log(LogLevel.Debug)`. What do you think?

**Comment by pshemass** [comment]
@jadlr I released 0.2.1 with `error` and `throwable` methods let me know if that's works for you.

**Comment by jdegoes** [comment]
@pshemass Agreed, `logInfo`, `logDebug`, etc. One for every `LogLevel`.
